website/content/docs/job-specification/parameterized.mdx

```diff
@@ -152,9 +152,59 @@ job "email-blast" {
 }
 ```
 
+### Interactions with [`periodic`][periodic] 
+
+When trying to schedule a job that is both `parameterized` and `periodic`, an internal 
+hierarchy comes into play: A parametrized job should not be dispatched without 
+parameters by the periodic configuration, forcing the parameterized option to take
+precedence: Once the job is dispatched and given parameters, the 
+periodic configuration comes into play and a new jobs will be dispatched according 
+to the periodic configurations and with the parameters of the triggering job.
+
+For example a job with the following configuration will not trigger any new jobs 
+until it is dispatched at least once, and after that the dispatched child will 
+periodically trigger more children with the given parameters. 
+
+```hcl
+  periodic {
+    crons = [
+      "*/40 * * * * * *"
+    ]
+  }
+  parameterized {
+    payload       = "required"
+    meta_required = ["dispatcher_email"]
+    meta_optional = ["pager_email"]
+  }
+```
+
+The first job `batch/periodic/parameterized` corresponds to the parameterized and
+periodic job, but it won't result in any new allocations until a `dispatch` with the 
+necessary parameters is submitted. The second job `batch/periodic` will trigger 
+the subsequent jobs `batch`. Any new dispatch will trigger a new flow of new 
+periodic jobs with the corresponding parameters. 
+
+```
+$ nomad job status                                                      
+ID                                                     Type                            Submit Date
+sync                                                   batch/periodic/parameterized    2024-11-07T10:43:30+01:00 // Original submitted job
+sync/dispatch-1730972650-247c6e97                      batch/periodic                  2024-11-07T10:44:10+01:00 // First dispatched job with parameters A
+sync/dispatch-1730972650-247c6e97/periodic-1730972680  batch                           2024-11-07T10:44:40+01:00 // Cron job with parameters A
```

Comment:
whoa! I didn't notice these great `// comments` in the preview, because I didn't see that the little code block could scroll to the right; I only saw to the `Type` column.  I'm not sure what to suggest for that... maybe say above this to "Scroll to the right for descriptions of each job."?